draft-ietf-openpgp-pqc.md

```diff
@@ -303,15 +303,17 @@ based on these mathematical problems is endangered by the advent of quantum
 computers, there is a need to extend OpenPGP by algorithms that remain secure
 in the presence of quantum computers.
 
-Such cryptographic algorithms are referred to as post-quantum cryptography.
-The algorithms defined in this extension were chosen for standardization by the
+Such cryptographic algorithms are referred to as post-quantum cryptography.  The
+algorithms defined in this extension were chosen for standardization by the
 National Institute of Standards and Technology (NIST) in mid 2022
 {{NISTIR-8413}} as the result of the NIST Post-Quantum Cryptography
-Standardization process initiated in 2016 {{NIST-PQC}}. Namely, these are
-ML-KEM (formerly CRYSTALS-Kyber) as a Key Encapsulation Mechanism (KEM), a KEM
-being a modern building block for public-key encryption, and ML-DSA (formerly
-CRYSTALS-Dilithium) as well as SLH-DSA (formerly SPHINCS+) as signature
-schemes.
+Standardization process initiated in 2016 {{NIST-PQC}}. Namely, these are ML-KEM
+("Module Lattice Key Encapsulation Mechanism", submitted to the NIST process as
+"CRYSTALS-Kyber") as a Key Encapsulation Mechanism (KEM), a KEM being a modern
+building block for public-key encryption, and ML-DSA ("Module Lattice Digital
+Signature Algorithm", submitted to the NIST process as "CRYSTALS-Dilithium") as
+well as SLH-DSA ("Stateless Hash Digital Signature Algorithm", submitted to the NIST process as
+"SPHINCS+") as signature schemes.
```

Comment:
It's not clear to me why "submitted to the NIST process as…" needs to be here for any of these three schemes, particularly in the introduction.  This is historical data at this point, right?  and the schemes as submitted don't necessarily match the schemes as standardized or used here.

If the goal is that someone searching an archive of IETF documents for "Dilithium", etc can find this draft, maybe just create a "historical notes" appendix or something.  otherwise, i think just dropping the "submitted to the NIST process as …" texts would also be fine.

Reply:
I agree, that this is information is superfluous in the introduction. I replaced it by the references to the FIPS standards for each scheme.